docs/attributes-registry/db.md

```diff
@@ -1,235 +1,204 @@
 
-**[3]:** Query parameters should only be captured when `db.query.text` is parameterized with placeholders.
+**[3]:** If setting a `db.mssql.instance_name`, `server.port` is no longer required (but still recommended if non-standard).
+
+**[4]:** In some SQL databases, the database name to be used is called "schema name". In case there are multiple layers that could be considered for database name (e.g. Oracle instance name and schema name), the database name to be used is the more specific layer (e.g. Oracle schema name).
+
+**[5]:** Query parameters should only be captured when `db.query.text` is parameterized with placeholders.
 If a parameter has no name and instead is referenced only by index, then `<key>` SHOULD be the 0-based index.
 
+`db.cassandra.consistency_level` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value          | Description | Stability                                                        |
+| -------------- | ----------- | ---------------------------------------------------------------- |
+| `all`          | none        | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `each_quorum`  | none        | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `quorum`       | none        | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `local_quorum` | none        | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `one`          | none        | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `two`          | none        | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `three`        | none        | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `local_one`    | none        | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `any`          | none        | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `serial`       | none        | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `local_serial` | none        | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
 `db.client.connections.state` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
 
-| Value  | Description | Stability |
-|---|---|---|
-| `idle` | idle | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `used` | used | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| Value  | Description | Stability                                                        |
+| ------ | ----------- | ---------------------------------------------------------------- |
+| `idle` | none        | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
oops?

Reply:
Oof, yeah, that's a regerssion, will fix, need to see why this happened here but not in build_tools

Reply:
https://github.com/open-telemetry/weaver/pull/131